gigascience/gigadb-website
# Remove sort from curation log table

# Pull request for issue: #1572

This page contians a table with logs: http://gigadb.gigasciencejournal.com:9170/adminDataset/update/id/5

![image](https://github.com/gigascience/gigadb-website/assets/143437854/38e9f39d-0266-4dc0-a47d-ad0239229ab3)

(might need to manually add elements to see content in the table)

the sortable table headings presented accessibility issues (loss of focus on sorting, sorting not actually working), so as per the comments on #1572 it was decided to remove sorting for this table

## How to test?

* Navigate to above page and verify that table is not sortable and nothing happens when clicking on a table header (no page refresh)

## How have functionalities been implemented?

`enableSorting` set to true (https://www.yiiframework.com/doc/api/1.1/CBaseListView#enableSorting-detail)

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
